### PR TITLE
Client app

### PR DESCRIPTION
**Pull request by ilyapuchka**
This adds a basic status bar client app that updates PR queue whenever user opens its popup.
Heavily based on raywenderlich.com tutorials.

Future TODO:
push notifications from service to update queue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,12 @@ Depending on the workflow of each team some of them may be disabled to suit thei
 👷‍♀️ **WIP, come back later** 👷‍♂️
 
 ![](https://media.giphy.com/media/26ybvJNaZZKpPONEc/giphy.gif)
+
+### Client app
+
+To install the client app:
+ - build it in Xcode and copy to your applications directory
+ - run `defaults write com.babylonhealth.WallEView Host <applicaiton address>`
```
Comment (AliSoftware) [inline README.md]:
```suggestion
 - run `defaults write com.babylonhealth.WallEView Host <application address>` 
```

File: WallEView/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        if popover.isShown {
+            closePopover(sender: sender)
+        } else {
+            showPopover(sender: sender)
+        }
+    }
+
+    func showPopover(sender: Any?) {
+        if let button = statusItem.button {
+            let vc: ViewController? = NSStoryboard.main.viewController()
```
Comment (AliSoftware) [inline WallEView/AppDelegate.swift:49]: 😱

File: WallEView/EventMonitor.swift [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    stop()
+  }
+
+  public func start() {
+    monitor = NSEvent.addGlobalMonitorForEvents(matching: mask, handler: handler)
+  }
+
+  public func stop() {
+    if monitor != nil {
+      NSEvent.removeMonitor(monitor!)
```
Comment (AliSoftware) [inline WallEView/EventMonitor.swift]: `if let`?

**Review by AliSoftware — APPROVED** [review]
👏

File: WallEView/AppDelegate.swift (hunk 2 above)
Comment (ilyapuchka) [reply WallEView/AppDelegate.swift:49]: 🤷‍♂ doing it in one expression with assignment does not set a generic type right (I guess extra `()` would solve it though)

Comment (AliSoftware) [reply WallEView/AppDelegate.swift:49]: I was mostly 😱(and trolling) at the usage of IB 😜

Comment (ilyapuchka) [reply WallEView/AppDelegate.swift:49]: Couldn't use SwiftUI because I'm not on Catalina =/

Comment (ilyapuchka) [reply WallEView/AppDelegate.swift:49]: TODO: use SwiftUI

**Review by mluisbrown — APPROVED** [review]

**Pull request merged**